### DISCUSSION TITLE
swagger generate doc from go file for file in formData

### DISCUSSION THREAD

**Discussion by ly269nva** [go-swagger/go-swagger#2814] [General]
## Problem statement
怎么用 go-swagger 定义表单中的上传文件字段，
how can i use go-swagger generate doc from go source file comment for a field of file in  formData?
 i want the result such this ( for api-fox):
![图片](https://user-images.githubusercontent.com/31555480/187212219-69806843-6867-4cf4-be75-26bee8b82e16.png)
i need the filetest field,  not file3 or other.
swagger.json such this 
![图片](https://user-images.githubusercontent.com/31555480/187212025-c144ae87-63ec-40b1-9174-09dde139cc3d.png)
"parameters": [{"name": "filet",
     "in": "formData",
		"format": "binary",
		"type": "string"
},{
		"type": "file",
		"description": "压缩包test2",
		"name": "filetest2",
		"in": "formData"
}]

## Steps to reproduce
this my swagger/docs/file.go file
[file-swagger.txt](https://github.com/go-swagger/go-swagger/files/9445012/file-swagger.txt)
then exec command `swagger generate spec -o swagger.json`
but i get the swagger.json belown
[swagger.txt](https://github.com/go-swagger/go-swagger/files/9445042/swagger.txt)
it cant not generate the param like filetest

## Environment
swagger version: v0.29.0
go version: go1.18.2
OS:   windows10

**Comment by casualjim** [comment]
take a look here: https://github.com/go-swagger/go-swagger/blob/master/fixtures/goparsing/classification/operations/noparams.go#L28-L33

**Reply by ly269nva** [reply]
ok, thank you

**Comment by rickyManalo** [comment]
What should my struct look like if the file upload needs to be a part of a model with other native fields? Should it be like https://github.com/go-swagger/go-swagger/blob/master/fixtures/goparsing/classification/operations/noparams.go#L52-L58? Should every non-file upload file also have // in: formData? 

Also, when I use formData, serving swagger.json doesn't show the parameters of the endpoint that have in: formData
